src/generic/generic-component-list-item/generic-component-list-item.twig

```diff
@@ -70,11 +70,13 @@
       </div>
     {% endif %}
     <div class="ecl-list-item__body">
-      <div class="ecl-list-item__meta">
-        {% block meta_block %}
-          {% include '@ecl/generic-component-meta' with { metas: _metas } only %}
-        {% endblock %}
-      </div>
+      {% if _variant not in ['navigation'] %}
```

Comment:
'default' is gone? if you compare only one value, why not `{% if _variant  == 'navigation' %}` ?